detectron2/utils/events.py

```diff
@@ -478,9 +497,282 @@ def clear_images(self):
         """
         self._vis_data = []
 
+    def clear_predictions(self):
+        """
+        Delete all predictions from `predictions` list.
+        """
+        self._predictions = []
+
     def clear_histograms(self):
         """
         Delete all the stored histograms for visualization.
         This should be called after histograms are written to tensorboard.
         """
         self._histograms = []
+
+
+class WandbWriter(EventWriter):
+    """
+    Write all scalars to a wandb tool.
+    """
+
+    def __init__(self, cfg: CfgNode, window_size: int = 20, **kwargs):
```

Comment:
As an alternative to passing `entity` in the config, could we make a note in the docstring that `entity` can be passed as a kwarg to `WandbWrite`?

Reply:
yes users can pass it in **kwargs